src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -29,13 +30,15 @@ public class AddCommand extends Command {
             + PREFIX_ROOMNUMBER + "ROOM NUMBER "
             + PREFIX_TELEGRAM + "TELEGRAM "
             + PREFIX_BIRTHDAY + "BIRTHDAY "
+            + "[" + PREFIX_FREETIMETAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ROOMNUMBER + "sw-01-01 "
             + PREFIX_TELEGRAM + "johnDoe "
-            + PREFIX_BIRTHDAY + "12/12/2000 ";
+            + PREFIX_BIRTHDAY + "12/12/2000 "
+            + PREFIX_FREETIMETAG + "Mon:1pm-2pm";
```

Comment:
Maybe we can show for multiple days? To show the user that they can set timings for multiple days upon creation.

Reply:
Yeap sounds good!